integration/tests/cook/test_basic.py

```diff
@@ -2217,6 +2217,10 @@ def test_user_usage_basic(self):
             util.kill_jobs(self.cook_url, [job_uuid])
 
     def test_user_usage_grouped(self):
+        settings_dict = util.settings(self.cook_url)
+        # If we're grouping quota, then we're doing HoL blocking mitigation, so this test is broken.
+        if len(settings_dict["quota-grouping"]) > 0:
+            self.skipTest("No pool match rules defined")
```

Comment:
Prefer `@unittest.skipIf`?

Reply:
Can't easily as we want the settings_dict variable, and we need the self.cook_url to get it, which I don't believe is available as a decorator.